app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -196,6 +199,24 @@ class MainViewModel @Inject constructor(
     }
     //endregion
 
+    //region Current Thread
+    val currentThreadUid = MutableLiveData<String?>(null)
+
+    val isInThread get() = currentThreadUid.value != null
+    val closeThreadTrigger = SingleLiveEvent<Unit>()
+
+    val downloadAttachmentsArgs = SingleLiveEvent<Triple<String, String, AttachmentType>>()
+    val newMessageArgs = SingleLiveEvent<NewMessageActivityArgs>()
+    val replyBottomSheetArgs = SingleLiveEvent<Pair<String, Boolean>>()
+    val threadActionsBottomSheetArgs = SingleLiveEvent<Triple<String, String, Boolean>>()
+    val messageActionsBottomSheetArgs = SingleLiveEvent<MessageActionsArgs>()
```

Comment:
We should create dataclass to encapsulate each args, put them in a separate file

Reply:
Will be done in another PR.